src/handlers/wildcard/unassign.ts

```diff
@@ -52,7 +52,7 @@ const checkBountyToUnassign = async (issue: Issue): Promise<boolean> => {
   const curTimestamp = new Date().getTime();
   const lastActivity = await lastActivityTime(issue, comments);
   const passedDuration = curTimestamp - lastActivity.getTime();
-  const pullRequest = await getOpenedPullRequestsForAnIssue(issue.number, issue.assignee.login);
+  const pullRequest = await getOpenedPullRequestsForAnIssue(issue.number, issue.assignee.login, "ready");
```

Comment:
can you use `gitIssueParser` or something instead of `getOpenedPullRequestsForAnIssue`?

Reply:
I see `gitIssueParser` returns true if any linked pr exists and false if not. And it is used nowhere. Since `getOpenedPullRequestsForAnIssue` is modified to get all kinds of prs, I am not sure if I have to tailor `gitIssueParser` to get all prs and use it...

Reply:
@pavlovcik I think we can use `getOpenedPullRequestsForAnIssue` for now